src/aioquic/tls.py

```diff
@@ -1838,12 +1854,20 @@ def _server_handle_hello(
                 )
 
             # send certificate verify
-            signature = self.certificate_private_key.sign(
-                self.key_schedule.certificate_verify_data(
-                    b"TLS 1.3, server CertificateVerify"
-                ),
-                *signature_algorithm_params(signature_algorithm),
-            )
+            if isinstance(self.certificate_private_key, ed25519.Ed25519PrivateKey):
```

Comment:
Instead of adding this if / else, couldn't you adjust the `signature_algorithm_params` method?